src/examples/java/com/amazonaws/crypto/examples/masterkeyprovider/multi/AwsKmsWithEscrow.java

```diff
@@ -0,0 +1,154 @@
+ * by combining a KMS master key with a raw RSA master key.
+ * <p>
+ * https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#master-key-provider
+ * <p>
+ * For more examples of how to use the KMS master key provider, see the
+ * 'masterkeyprovider/awskms' examples'
+ * <p>
+ * For more examples of how to use the raw RSA master key, see the
+ * see the 'masterkeyprovider/rawrsa' examples.
+ * <p>
+ * In this example we generate a RSA keypair
+ * but in practice you would want to keep your private key in an HSM
+ * or other key management system.
+ * <p>
+ * In this example, we use the one-step encrypt and decrypt APIs.
+ */
+public class AwsKmsWithEscrow {
+
+    /**
+     * Demonstrate configuring a master key provider to use an AWS KMS CMK and a RSA wrapping key.
+     *
+     * @param awsKmsCmk       The ARN of an AWS KMS CMK that protects data keys
+     * @param sourcePlaintext Plaintext to encrypt
+     */
+    public static void run(final AwsKmsCmkId awsKmsCmk, final byte[] sourcePlaintext) throws GeneralSecurityException {
+        // Instantiate the AWS Encryption SDK.
+        final AwsCrypto awsEncryptionSdk = new AwsCrypto();
+
+        // Prepare your encryption context.
+        // https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#encryption-context
+        final Map<String, String> encryptionContext = new HashMap<>();
+        encryptionContext.put("encryption", "context");
+        encryptionContext.put("is not", "secret");
+        encryptionContext.put("but adds", "useful metadata");
+        encryptionContext.put("that can help you", "be confident that");
+        encryptionContext.put("the data you are handling", "is what you think it is");
+
+        // Generate an RSA key pair to use with your master key.
+        // In practice, you should get this key from a secure key management system such as an HSM.
+        final KeyPairGenerator kg = KeyPairGenerator.getInstance("RSA");
+        // The National Institute of Standards and Technology (NIST) recommends a minimum of 2048-bit keys for RSA.
+        // https://www.nist.gov/publications/transitioning-use-cryptographic-algorithms-and-key-lengths
+        kg.initialize(4096);
+        final KeyPair keyPair = kg.generateKeyPair();
+
+        // Create the encrypt master key that only has access to the public key.
+        final JceMasterKey escrowEncryptMasterKey = JceMasterKey.getInstance(
+                keyPair.getPublic(),
+                null,   // Private key is null
+                // The provider ID and key ID are defined by you
+                // and are used by the raw RSA master key
+                // to determine whether it should attempt to decrypt
+                // an encrypted data key.
+                "some managed raw keys",  // provider corresponds to key namespace for keyrings
+                "my RSA wrapping key",      // key ID corresponds to key name for keyrings
```

Comment:
Depending on preference, let's either line these up or use a consistent spacing between the code and comment.

```suggestion
                "some managed raw keys",  // provider corresponds to key namespace for keyrings
                "my RSA wrapping key",    // key ID corresponds to key name for keyrings
```